CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # osquery Changelog
 
+
+<a name="5.2.1"></a>
+## [5.2.1](https://github.com/osquery/osquery/releases/tag/5.2.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.1.0...5.2.1)
+
+Osquery 5.2.1 brings native m1 support to the macOS platform. 
+
+Representing commits from 9 contributors! Thank you all.
+
+### New Features
+
+- Apple Silicon support ([#7330](https://github.com/osquery/osquery/pull/7330))
+
+### Table Changes
+
+- Update `time` table to reflect UTC values ([#7276](https://github.com/osquery/osquery/pull/7276))
+- Hide the deprecate `antispyware` column in `windows_security_center` ([#7411](https://github.com/osquery/osquery/pull/7411))
+
+### Under the Hood improvements
+
+
+### Bug Fixes
+
+
```

Comment:
Fixed by the Apple Silicon PR 
 - https://github.com/osquery/osquery/issues/7359 SignatureTest no longer gets null results for test_get_unsigned on M1 Macs
 - https://github.com/osquery/osquery/issues/7360 Resolved failing tests_integration_tables-test on M1 Macs

Reply:
Hrm. You're right to call them out, but I think this is the wrong place. Or at least, it's not how I think of this. I think of these as a record/explanation of changes. Where-as these are bugs in m1, that are implied as fixed by the m1 support.

Probably the right thing is to make the intro paragraph a bit swankier. I'll give that a shot